presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/crux/Crux.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.analyzer.crux;
```

Comment:
How about moving all semantic tree related concepts to `com.facebook.presto.sql.analyzer.crux.tree`?

Then we can leave this class and other executable classes in `com.facebook.presto.sql.analyzer.crux`.

Reply:
I don't want to change too many things, it would make syncing future changes harder.